### PR TITLE
update(deps): storybook and types

### PR DESCRIPTION
**Pull request by allfayn**
Changes:
- update all main packages except React
- husky will start prettier on pre_commit hook and lint on pre_push
- prettier for scss
- move build and chromatic publish to Github Action 
- publish chromatic only when push to master
- new function for act in test files
- fix types errors

### CODE REVIEW

File: packages/ui/src/Terminal.tsx [hunk 1]
```diff
@@ -92,9 +92,9 @@ export const Terminal: FC<TerminalProps> = memo(
 
         addExecutionHistoryItem({ command: inputVal, res, prompt })
       } catch (err) {
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-        const errorTyped: Error = err
-        addExecutionHistoryItem({ command: inputVal, error: errorTyped.message, prompt })
+        if (err instanceof Error) {
```
Comment (fantkolja) [inline packages/ui/src/Terminal.tsx:95]: What other types of `err` could it be? Is it OK that we just ignore them?

File: package.json [hunk 2]
```diff
@@ -16,7 +16,8 @@
     "husky": "^4.3.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^26.5.2",
-    "lerna": "^3.22.1",
+    "lerna": "^4.0.0",
```
Comment (fantkolja) [inline package.json:19]: Getting the following error after installing deps and running `npm run start in the `/ui` folder
image

**Review by fantkolja — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (allfayn) [reply package.json:20]: fixed

File: packages/ui/src/Terminal.tsx (hunk 1 above)
Comment (allfayn) [reply packages/ui/src/Terminal.tsx:95]: It's the base type of Error. I think we catch them all in this situation. And have a certain type for needed data

**Review by fantkolja — CHANGES_REQUESTED** [review]
Nice work!
Few comments left

File: packages/ui/styles/fonts.css [hunk 3]
```diff
@@ -1,41 +1,32 @@
...
   font-style: normal;
   font-weight: 500;
   src: url('../fonts/roboto-v27-latin-500.eot'); /* IE9 Compat Modes */
-  src: local(''),
-  url('../fonts/roboto-v27-latin-500.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
-  url('../fonts/roboto-v27-latin-500.woff2') format('woff2'), /* Super Modern Browsers */
-  url('../fonts/roboto-v27-latin-500.woff') format('woff'), /* Modern Browsers */
-  url('../fonts/roboto-v27-latin-500.ttf') format('truetype'), /* Safari, Android, iOS */
-  url('../fonts/roboto-v27-latin-500.svg#Roboto') format('svg'); /* Legacy iOS */
+  src: local(''), url('../fonts/roboto-v27-latin-500.eot?#iefix') format('embedded-opentype'),
```
[the remark this answers is not in the document]
Comment (allfayn) [reply packages/ui/styles/fonts.css:29]: removed

File: packages/ui/styles/fonts.css [hunk 4]
```diff
@@ -1,41 +1,32 @@
...
+    url('../fonts/roboto-v27-latin-500.woff2') format('woff2'), url('../fonts/roboto-v27-latin-500.woff') format('woff'),
+    url('../fonts/roboto-v27-latin-500.ttf') format('truetype'), url('../fonts/roboto-v27-latin-500.svg#Roboto') format('svg'); /* Legacy iOS */
```
[the remark this answers is not in the document]
Comment (allfayn) [reply packages/ui/styles/fonts.css:25]: removed eot files

File: packages/ui/package.json [hunk 5]
```diff
@@ -45,68 +46,77 @@
...
+    "stylelint-prettier": "^2.0.0",
+    "webpack": "^5.65.0",
+    "yup": "^0.32.11"
   },
   "peerDependencies": {
     "react": "^16.14.0 || ^17.0.0",
-    "react-dom": "^16.14.0 || ^17.0.0"
+    "react-dom": "^16.14.0 || ^17.0.0",
+    "@types/enzyme": "^3.10.10",
+    "enzyme": "^3.11.0"
```
[the remark this answers is not in the document]
Comment (allfayn) [reply packages/ui/package.json:120]: Yes, removed from `peerDependencies`

File: packages/ui/__tests__/InteractiveListFormik.test.tsx [hunk 6]
```diff
@@ -40,11 +40,13 @@ describe('InteractiveListFormik', () => {
 
     // eslint-disable-next-line @typescript-eslint/require-await
     await act(async () => {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
       simulateChange(wrapper.find('input'), '127.0.0.1')
     })
 
     // eslint-disable-next-line @typescript-eslint/require-await
     await act(async () => {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-member-access
```
[the remark this answers is not in the document]
Comment (allfayn) [reply packages/ui/__tests__/InteractiveListFormik.test.tsx:49]: fixed, remove ignore line

**Review by fantkolja — APPROVED** [review]

**Review by utkukaratas — APPROVED** [review]

**Pull request merged**